src/main/java/frc/robot/RobotContainer.java

```diff
@@ -9,6 +9,7 @@
 import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
```

Comment:
Why was this added when it is not used? While you are at it, please remove the other unused imports.